extensions/rocksdb-repos/DatabaseContentRepository.cpp

```diff
@@ -41,14 +42,17 @@ bool DatabaseContentRepository::initialize(const std::shared_ptr<minifi::Configu
   } else {
     directory_ = configuration->getHome() + "/dbcontentrepository";
   }
-  rocksdb::Options options;
-  options.create_if_missing = true;
-  options.use_direct_io_for_flush_and_compaction = true;
-  options.use_direct_reads = true;
-  options.merge_operator = std::make_shared<StringAppender>();
-  options.error_if_exists = false;
-  options.max_successive_merges = 0;
-  db_ = utils::make_unique<minifi::internal::RocksDatabase>(options, directory_);
+  auto db_opts = [] (internal::Writable<rocksdb::DBOptions>& db_opts) {
```

Comment:
Nitpicking: as these are going to be functions we call later, I would prefer them to be named accordingly. "set_db_opts" for eg.